README.adoc

```diff
@@ -1330,7 +1330,7 @@ Use the short form of the I18n methods: `I18n.t` instead of `I18n.translate` and
 
 === Lazy Lookup [[lazy-lookup]]
 
-Use "lazy" lookup for the texts used in views. Let's say we have the following structure:
+Use "lazy" lookup for locale entries from views and controllers. Let's say we have the following structure:
```

Comment:
Helpers too?

Reply:
I believe this may lead to confusion.
The term "helpers" can be used for ["form helpers"](https://guides.rubyonrails.org/form_helpers.html), ["view helpers"](https://guides.rubyonrails.org/action_view_helpers.html), [ad-hoc helpers](https://github.com/spree/spree/blob/main/core/app/helpers/spree/locale_helper.rb#L17), and even [email helpers](https://github.com/discourse/discourse/blob/bebb4fd11181f3deb277b2e4cb0e7148b61fb71d/app/helpers/email_helper.rb#L20). I have no certainty that it is possible to use lazy lookup in all those cases.